scripts/check-users-in-space-developer-role.ps1

```diff
@@ -1,87 +1,126 @@
-#!/usr/bin/env pwsh
```

Comment:
Can we leave that so it works on unix directly

Reply:
Reinstated